engine.py

```diff
@@ -55,22 +55,42 @@ def fight(self):
         After attack/support it changes the team to another one.
         :return: Stops attack if chosen character is stunned
         """
+        # # working version
```

Comment:
Git jest po to, żeby tego tak nie robić :p

Reply:
zeby jak i czego nie robic?
chodzi Ci o zmiane teamu po wykonaniu ruchu, o czym juz mowilismy kiedys?